usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -74,10 +74,13 @@ def handle_loading(self, db_cursor, *args, **options):
         self.skip_final_of_fy_calculation = options["skip_final_of_fy_calculation"]
         self.db_cursor = db_cursor
 
+        logger.info(f"Starting processing for submission {self.submission_id}...")
```

Comment:
Is there more information on the submission at this point? I realized yesterday I missed the agency name in the log output (originally printed during the ID fetch from Broker, but included the sub load starts would be more helpful)

Reply:
> but included the sub load starts would be more helpful

Having a hard time parsing this.  Can you use smaller words that my sleep deprived brain my better comprehend?

Reply:
LOL.

(originally _the names were_ printed during the ID fetch from Broker, but _including them_ at _submission_ load starts would be more helpful)

Reply:
Couldn't add it this early, but it should appear near the top of the logs now.

Reply:
![thanks a bunch](https://media.giphy.com/media/wNJWERozDsIrm/giphy.gif)